torch/csrc/utils/python_dispatch.cpp

```diff
@@ -192,10 +193,10 @@ static torch::_RegisterOrVerify register_or_verify() {
 static py::object ophandle_call_boxed(
     const c10::OperatorHandle& handle,
     py::args args,
-    py::kwargs kwargs) {
+    const py::kwargs& kwargs) {
   auto stack = torch::jit::createStackForSchema(
       handle.schema(),
-      args,
+      std::move(args),
```

Comment:
Hmm, only args should be moved by value? Is this intentional? I guess we just iterate through the kwargs dict then?

Reply:
kwargs is used as const arguments in the function body, no benefits to move